onfido-java/src/test/java/com/onfido/integration/ApiIntegrationTest.java

```diff
@@ -1,26 +1,81 @@
 package com.onfido.integration;
 
 import java.io.IOException;
+import com.onfido.Onfido;
+import com.onfido.Onfido.Builder;
+import com.onfido.JsonObject;
+
+import static org.junit.Assert.assertEquals;
+
 import okhttp3.mockwebserver.MockResponse;
 import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeTest;
 
 class ApiIntegrationTest {
+
+  private String apiToken = System.getenv("ONFIDO_API_TOKEN");
+  private Onfido.Builder builder = Onfido.builder();
+  protected Onfido onfido;
+
+  private RecordedRequest request;
   private MockWebServer server;
 
-  @BeforeTest
-  public void clearServer() {
-    server = null;
+  protected ApiIntegrationTest() {
+    if ( ! mockingEnabled() )
+    {
+      onfido = builder.apiToken(apiToken).regionEU().build();
+    }
   }
 
+  @BeforeTest
+  private void setup() {}
+
   @AfterTest
-  public void shutdownServer() throws IOException {
-    if (server != null) {
+  private void tearDown() throws IOException {
+    if ( mockingEnabled() && server != null )
+    {
       server.shutdown();
     }
   }
```

Comment:
Coding style could be improved as well ✨ 

```suggestion
    if (mockingEnabled() && server != null) {
      server.shutdown();
    }
  }
```